docs/README.md

```diff
@@ -109,6 +109,30 @@ After starting TWCManager, the script will run in the foreground and will regula
    * The line below this reports the same values but in amps instead of watts.
    * The final line shows the minAmpsPerTWC value, which is the minimum number of amps that the master must offer each slave before we tell the attached vehicle to charge (via the Tesla API).
 
+## Running TWCManager as a Service
+
+The following commands make TWCManager run automatically (as a service) when the Raspberry Pi boots up:
+
+Install the service
+```
+sudo systemctl enable /home/pi/TWCManager/twcmanager.service
+```
+
+Start the TWCManager service. This will  persist after rebooting.
+
+```
+sudo systemctl start twcmanager
+```
+
+To check the output of the TWCManager service
+```
+journalctl -f
+```
+
+To stop the TWCManager service permanently.  This will persist after rebooting.
```

Comment:
the stop command just stops the service, it doesn't disable it. Use `systemctl disable` for that

Reply:
Thanks for the detailed comments - not an expert on these services so changes implemented in the pull request.